src/rpc/RPC.ml

```diff
@@ -38,8 +40,11 @@ let handle_autofix dryrun edits =
     in
     (List.length modified_files, [])
 
-let handle_sarif_format caps hide_nudge engine_label (rules : fpath)
-    (cli_matches : cli_match list) (cli_errors : cli_error list) =
+let handle_sarif_format caps hide_nudge engine_label show_dataflow_traces
+    (rules : fpath) (cli_matches : cli_match list) (cli_errors : cli_error list)
+    =
+  (* We need to parse the rules here, so we need to init the parsers. *)
```

Comment:
hmm, this thing is a bit fragile; maybe it's better if we do that in the main entry point of RPC.ml instead of in each call.